clients/drcachesim/simulator/cache_simulator.cpp

```diff
@@ -363,6 +363,15 @@ cache_simulator_t::cache_simulator_t(std::istream *config_file)
         success_ = false;
         return;
     }
+    // For larger hierarchies, especially with coherence, using hashtables
```

Comment:
Why is it that we don't enable hashtables through the other constructor which takes knobs instead?

Reply:
It is not possible to configure more than 2 levels of caches with knobs, so, at least according to my local runs, hashtables would not be a win for any knob config.